### ISSUE TITLE
Declare that "This is a community effort"

### ISSUE THREAD

**Issue by davidak** [NixOS/nixos-homepage#284]
Just add a sentence to the "About the community" section on the homepage.

https://nixos.org/

To make clear that NixOS is not a commercial product of a for-profit company.

**Comment by vcunat** [comment]
I'm not sure how to formulate this well.  Maybe something like this?
```diff
-NixOS is a Linux distribution
+NixOS is a community-driven Linux distribution
```

Notably, a non-trivial part of the active contributors apparently **do** make money thanks to NixOS.  I personally do not see a problem in that at all &ndash; actually it should make their contributions more sustainable.  So far I haven't seen significant "political" issues in the community.

**Comment by chreekat** [comment]
It might be good to clarify the legal status of whoever/whatever owns the NixOS trademark and domain names before making such a claim. :) You'd probably also want to clarify the governance structure of that entity to back up the "non-profit" part.

The *code bases* are community efforts by virtue of being freely licensed, but that is a small piece of the whole pie.

**Comment by davidak** [comment]
>It might be good to clarify the legal status of whoever/whatever owns the NixOS trademark and domain names before making such a claim.

Good point. I hope the [NixOS Foundation](https://nixos.org/nixos/foundation.html). @edolstra 

We should mention it there. It's currently quiet hard to find on the website.

---

Nix, NixOS and other related projects are developed by a diverse community from all over the world. The ecosystem's infrastructure is managed and supported by the [NixOS Foundation](https://nixos.org/nixos/foundation.html), a non-profit organisation registered in The Netherlands. If you are a happy user, please consider [donating](https://nixos.org/nixos/foundation.html)!

We should also describe the organization structure on the website, like the RFC Steering Committee, Nixpkgs Maintainers (New GitHub group) etc. But that don't fit into this section, so we probably need a new page. Maybe the Foundation could be just one section on that page.

**Comment by chreekat** [comment]
Aha! Thanks @davidak, I didn't even know. +1 to your description and link.
